app/Http/Controllers/RegisterController.php

```diff
@@ -57,7 +57,22 @@ public function loginUser(Request $request){
                 }
     }
 
-    public function dashboard(Accountant $accountant){
-        return view('dashboard.userListing');
+    function dashboard()
+    {
+        return view('dashboard.welcome');
+    }
+
+    public function list(){
+        $accountant_data = Accountant::all();
+        return view('dashboard.userListing', ['users' => $accountant_data]);
+    }
+
+    function logout()
+    {
+        if(session()->has('loginId'))
```

Comment:
```suggestion
        if(session()->has('login_id'))
```
All `array keys` or `object properties` should be in `snake_case` only, update similar changes on other files too.

Reply:
PR updated as per reviews.